docs/pywbemcli/commands.rst

```diff
@@ -787,6 +788,109 @@ Valid output formats are :term:`CIM object output formats` or
 
 See :ref:`pywbemcli instance query --help` for the exact help output of the command.
 
+.. _`Instance shrub command`:
+
+Instance shrub command
+^^^^^^^^^^^^^^^^^^^^^^
+
+The ``instance shrub`` command executes a set of requests to get the
+association relationships for a non-association CIM instance defined by
+INSTANCENAME in a namespace and displays the result either as tree in ascii
+or as a table showing the roles, reference classes, associated
+classes and associated instances for the input instance.
+
+A shrub is a structure that attempts to show all of the relationships and the
+paths between the input INSTANCENAME and the associated instances whereas the
+References command only shows reference classes or instances and the
```

Comment:
Change "reference classes or instances" to "referencing classes or instances".

Reply:
Changed but to association(reference) rather than referencing.  I have badly conflated the two terms for years now but trying to use association and associated as the names for those components.

Reply:
DONE